pybossa/core.py

```diff
@@ -581,6 +581,12 @@ def _unauthorized(e):
                         description=UNAUTHORIZED)
         return handle_content_type(response)
 
+    @app.errorhandler(423)
+    def _locked(e):
```

Comment:
This method likely needs a unit test to increase code coverage. Consider calling `_locked()` to ensure no exception raised.